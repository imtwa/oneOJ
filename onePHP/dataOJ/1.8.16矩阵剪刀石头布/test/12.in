46 12 87
RRPSPRSPSSPP
RRSRPRRSPSRS
RSSSRSRPSRPR
RRRRRSSPRPPS
RRRSSSRSPRRP
SRPPSSRRRSRS
RSSRRSPPRRRR
PSRSSSPRPPPS
PSPPSSSRPPSS
SRSRRRSRPPPR
SSRRSPSSSPSR
PSRSPPPSPRPS
SSRSSSSPRPPP
RPPSSPPRRSSS
RSSPPSSPRPSP
PSPRRRRSSRRR
PPPRPSRPSRSR
PSSPSRSPSRSR
PRPSRSSRPSRS
PRRRRPRPRRRR
RSSRSPPPSSSP
SRRPRSRSPRPP
PSRRRSSSRPPP
SRPRRSSSRPPR
SRPPRPSRPSPR
PSRSRSSRPSRP
RPPSSSSSPSPS
SRSSRRSRRRPP
PRPPRPRPPPSS
PRPRRRSRRPPP
PSRPSSRSPSRS
PSRPRRRSRPSR
PPPPPPSRSPSR
RPRSSPPRRSPP
SSPRPRRSPPPR
RRSPPPSPRRSR
RPRRRRPSPRSP
RSRPSSPSPPPR
RSSSPPSRRSSP
PRPSRRPPSPSS
SRSRRPPRSSPS
SRPSRSPSPRPP
PSSSSPSSRSPS
RSPPPSSPPPRR
SRPRPRPPSRPP
SRPSSSSRRPPR
