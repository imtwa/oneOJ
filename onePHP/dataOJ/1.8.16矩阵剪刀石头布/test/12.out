RPSPRRPRRSPP
SRRSRPSPPRRS
SRPRPSRPSPRS
RPSPRPSPRSPP
PSRSPRPRSPRR
SRPPPPSPPRSP
RSSRSPRSPRSP
PSRSPRSPRSSR
RPPPPPRRSPPS
RPSRSSPRRSPR
SRPSPRRSPRSS
PSRPRSSPRSPP
SSRRSPPRSSPR
RPPSPRRSPRSP
SPRPRRSPPSSP
SRSRSPRSRPRR
SRSSSRRPSRSS
RPRRRSRSPSPS
SSRPPRPPPSSR
PPSSSRSSPRRP
RPSRSPRRRPRS
PSRPRSSPRSSP
SRPSPRSPSPPR
RPSRSPRSPPRS
PSRPRRSPPSPR
PSRSSPRSSRPR
RPPPPRSPRRSP
SRSPPRSPSPPS
PRPRSPRSPSSR
RSRSPRSPSRRP
RPSSRSPPRPSS
SRPPRSPSRPSR
PSPSPRPSRSRP
PSPRSPRRPRRS
RRSPPRSSSRPR
RPRSPPSRSSSR
PSPPPSRPRRSP
RRPSRPPSPPPR
PPRRRPSRRSPR
SSRPSRPPSPRS
RPRRPSRRSSPR
RPSSRRPSPRRR
SRSSRPPSRSPP
PRSRPSPPPPRS
RPSSRRPPSRSP
PSRPPSSRSSPR
